hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
```

Comment:
Our convention is to add a final line to the release note for non-maintainers:  "Thanks to Tim [Martin] for this contribution".  The surname, and indeed whole thing, is optional but I'd encourage you to do it even if it feels a little awkward - we really do appreciate these contributions! :smile:

Reply:
Will do.